preprocessing/metrics.py

```diff
@@ -22,11 +63,22 @@ def _index(meta, plate_types, ignore_codes=None, include_codes=None):
     return index.values
 
 
-def _group_negcons(meta: pd.DataFrame):
-    """
-    Hack to avoid mAP computation for negcons. Assign a unique id for every
-    negcon so that no pairs are found for such samples.
+def _group_negcons(meta: pd.DataFrame) -> None:
```

Comment:
We may want to use the `assign_reference_index` function as in the [copairs example](https://github.com/cytomining/copairs/blob/main/examples/phenotypic_activity.ipynb) I'll open an issue for this.

Reply:
created: https://github.com/broadinstitute/jump-profiling-recipe/issues/26